pages/pricing.js

```diff
@@ -0,0 +1,135 @@
+                        <Image src={SecurityLockImg} alt="Security Lock image" />
+
+                        <p className="text-black py-3 text-lg md:text-xl lg:text-3xl xl:text-4xl xl:leading-[44px] font-Inter font-semibold">
+                          BASIC SECURITY
+                        </p>
+
+                        <div className="text-black pt-6 lg:pt-10 text-sm">
+                          <p>Unlimited transactions</p>
+                          <p>Network Connections </p>
+                          <p>Location tracking </p>
+                          <p>Real-time messaging</p>
+                        </div>
+                      </div>
+                    </div>
+                    <div className="mb-3 absolute bottom-8 justify-end flex ">
+                      <Button color="secondary" className="text-base lg:text-xl" size="xl">
+                        Get Started Now
+                      </Button>
+                    </div>
+                  </div>
+                </div>
+                <div className="h-[530px] lg:h-[600px] w-[320px] md:w-[300px] lg:w-[400px] xl:w-[445px] border border-warning-500 rounded-lg">
+                  <div className="border-y-4 flex flex-col items-center relative h-[530px] lg:h-[600px] border-warning-500 rounded-md">
+                    <div className="  flex flex-col items-center justify-between">
+                      <div className="text-center p-10 pt-14">
+                        <Image src={SecuritySafeImg} alt="Security Lock image" />
+
+                        <p className="text-warning-600 py-3 text-lg md:text-xl lg:text-3xl xl:text-4xl xl:leading-[44px] font-Inter font-semibold">
+                          PREMIUM SECURITY
+                        </p>
+
+                        <div className="text-black pt-6 lg:pt-10 text-sm">
+                          <p>Unlimited transactions</p>
+                          <p>Network Connections </p>
+                          <p>Location tracking </p>
+                          <p>Real-time messaging</p>
+                        </div>
+                      </div>
+                    </div>
+                    <div className="mb-3 absolute bottom-8 justify-end flex ">
+                      <Button color="warning" className="text-base lg:text-xl" size="xl">
+                        Coming Soon
+                      </Button>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div className=""></div>
+          </div>
+        </div>
+      </div>
+      <div className="bg-[#F5F6F8] px-14 py-16 lg:py-20 ">
+        <div>
+          <div className="mt-2  avatar-group text-center">
+            <div className="relative z-20 inline-flex items-center justify-center w-12 h-12 lg:w-14 lg:h-14 -ml-4  transition-all duration-200 border-2 border-white border-solid rounded-full ease-soft-in-out text-size-xs hover:z-40">
+              <img
```

Comment:
Can we use the normal NextJS image here or change these to local images?

Reply:
Fixed